diff -urN phablet-preview-9999/plugins/Unity/CMakeLists.txt phablet-preview-9999-patched/plugins/Unity/CMakeLists.txt
--- phablet-preview-9999/plugins/Unity/CMakeLists.txt	2013-05-28 17:31:26.310024771 +0930
+++ phablet-preview-9999-patched/plugins/Unity/CMakeLists.txt	2013-05-28 22:17:57.152143686 +0930
@@ -27,8 +27,8 @@
     )
 
 target_link_libraries(Unity-qml
-    ${QTDEE_LDFLAGS}
     ${UNITYCORE_LDFLAGS}
+    ${QTDEE_LDFLAGS}
     ${Qt5DBus_LIBRARIES}
     ${Qt5Gui_LIBRARIES}
     )
