diff -urN mir-9999/src/server/default_server_configuration.cpp mir-9999-patched/src/server/default_server_configuration.cpp
--- mir-9999/src/server/default_server_configuration.cpp	2013-03-15 11:27:24.966054000 +1030
+++ mir-9999-patched/src/server/default_server_configuration.cpp	2013-03-15 11:35:50.698596787 +1030
@@ -16,6 +16,8 @@
  * Authored by: Alan Griffiths <alan@octopull.co.uk>
  */
 
+#include <iostream>
+
 #include "mir/default_server_configuration.h"
 
 #include "mir/options/program_option.h"
diff -urN mir-9999/src/server/frontend/protobuf_message_processor.cpp mir-9999-patched/src/server/frontend/protobuf_message_processor.cpp
--- mir-9999/src/server/frontend/protobuf_message_processor.cpp	2013-03-15 11:27:24.966054000 +1030
+++ mir-9999-patched/src/server/frontend/protobuf_message_processor.cpp	2013-03-15 11:36:53.514599827 +1030
@@ -16,6 +16,8 @@
  * Authored by: Alan Griffiths <alan@octopull.co.uk>
  */
 
+#include <iostream>
+
 #include "protobuf_message_processor.h"
 #include "mir/frontend/resource_cache.h"
 
diff -urN mir-9999/src/server/frontend/socket_session.cpp mir-9999-patched/src/server/frontend/socket_session.cpp
--- mir-9999/src/server/frontend/socket_session.cpp	2013-03-15 11:27:24.966054000 +1030
+++ mir-9999-patched/src/server/frontend/socket_session.cpp	2013-03-15 11:37:09.129600583 +1030
@@ -16,6 +16,8 @@
  * Authored by: Alan Griffiths <alan@octopull.co.uk>
  */
 
+#include <iostream>
+
 #include "socket_session.h"
 
 #include "ancillary.h"
